### ISSUE TITLE
Could you provide the hyperparameters for 95.6% (highest 95.67) test accuracy?

### ISSUE THREAD

**Issue by YingHH1** [henryqin1997/CIFAR10-ResNet50-PyTorch#1]
Hi Henry,

I was not able to achieve 95.6% test accuracy using the command in run.sh

Could you provide the hyperparameters for 95.6% (highest 95.67) test accuracy?

Many thanks
